sites/fast-website/src/app/data/community.data.ts

```diff
@@ -0,0 +1,44 @@
+import DiscordIcon from "svg/icon-discord.svg";
+import GithubIcon from "svg/icon-github.svg";
+import TwitterIcon from "svg/icon-twitter.svg";
+import MediumIcon from "svg/icon-medium.svg";
+
+export interface CommunityContentPlacementData {
+    actionLink: string;
+    actionText?: string;
+    body?: string;
+    header?: string;
+    icon: string;
+}
+
+export const communityContentPlacementData: CommunityContentPlacementData[] = [
+    {
+        actionLink: "https://discord.gg/FcSNfg4",
+        actionText: "Join the Discord Chat",
```

Comment:
This may be better as "Join our Discord Chat" to be consistent in tone with "Follow us on Twitter"?